src/services/auth-service.ts

```diff
@@ -97,13 +100,7 @@ export const verifyRefreshToken = async (refreshToken: string): Promise<UserPayl
   return { userId, userRole };
 };
 
-export const authenticateLocal = (req: Request, res: Response): Promise<User | Error> =>
-  new Promise((resolve, reject) => {
-    passport.authenticate('local', { session: false }, (err: Error, user: User) => {
-      if (err) reject(err);
-      resolve(user);
-    })(req, res);
-  });
+export const authenticateLocal = passport.authenticate('local', { session: false });
```

Comment:
It doesn't seem to be used anymore... Please check

Reply:
Removed